sdk/core/azure-core-experimental/azure/core/experimental/transport/_pyodide.py

```diff
@@ -65,6 +79,11 @@ async def load_body(self) -> None:
 class PyodideStreamDownloadGenerator(AsyncIterator):
     """Simple stream download generator that returns the contents of
     a request.
+
+    :param pipeline: The pipeline object
+    :type pipeline: ~azure.core.pipeline.Pipeline
```

Comment:
I don't see the typing for AsyncHttpXStreamDownloadGenerator?

Reply:
good catch, added it in